docs/usage.rst

```diff
@@ -0,0 +1,719 @@
+with a storage system.
+The class
+:class:`audbackend.FileSystem`
+implements
+:class:`audbackend.Backend`
+for a standard file system.
+
+
+We start by registering the class
+(in fact,
+the class is registered
+by default,
+but it doesn't hurt
+if we do it again).
+
+.. jupyter-execute::
+
+    import audbackend
+
+    audbackend.register('file-system', audbackend.FileSystem)
+
+
+We create an instance
+and provide two arguments:
+
+* ``host``: the host address,
+  in this case a folder on the local file system.
+* ``repository``: the repository name,
+  in this case a sub-folder within the ``host`` folder
+  (it is possible to have several repositories
+  on the same host).
+
+.. jupyter-execute::
+
+    backend = audbackend.create('file-system', './host', 'repo')
+
+
+This will create an empty repository
+(in our case the folder ``'./<host>/<repo>/'``).
+To access an existing repository,
+we would do:
+
+.. jupyter-execute::
+
+    backend = audbackend.access('file-system', './host', 'repo')
+
+
+To put a file on the backend,
+we provide two path arguments.
+
+* ``src_path``: path to a file on the local file system.
+  This is the file we want to store on the backend.
+* ``dst_path``: virtual path that represents the file on the backend.
+  It serves as an alias that is understood by all backends.
+
+With
+:mod:`audbackend`
+we can store different
+versions of a file.
+Hence,
+we attach a
+``version`` string
+to the backend path.
+Together they
+provide a unique identifier
+to the file.
+
+.. jupyter-execute::
+
+    import tempfile
+
+    with tempfile.TemporaryDirectory() as tmp:
+        src_path = os.path.join(tmp, '~')
+        with open(src_path, 'w') as fp:
+            fp.write('Hello world')
+        backend.put_file(src_path, '/file.txt', '1.0.0')
+
+
+We check if the file exists on the backend.
+
+.. jupyter-execute::
+
+    backend.exists('/file.txt', '1.0.0')
+
+
+And access its meta information.
+
+.. jupyter-execute::
+
+    backend.checksum('/file.txt', '1.0.0')
+
+.. jupyter-execute::
+
+    backend.date('/file.txt', '1.0.0')
+
+.. jupyter-execute::
+
+    backend.owner('/file.txt', '1.0.0')
+
+
+We fetch the file
+from the backend
+and verify it has
+the expected content.
+
+.. jupyter-execute::
+
+    path = backend.get_file('/file.txt', 'local.txt', '1.0.0')
+    with open(path, 'r') as fp:
+        display(fp.read())
+
+
+Then we modify it and
+publish it under a new version.
+
+.. jupyter-execute::
+
+    with open(path, 'a') as fp:
+        fp.write('. Goodbye!')
+    backend.put_file(path, '/file.txt', '2.0.0')
+
+
+It is possible to upload
+one or more files
+as an archive.
+Here,
+we compress the file
+we just modified
+and put it under
+the sub-path ``'/a/'``.
+
+.. jupyter-execute::
+
+    backend.put_archive('.', '/a/file.zip', '2.0.0', files=[path])
+
+
+When we get an archive from the backend
+we can automatically extract it.
+
+.. jupyter-execute::
+
+    paths = backend.get_archive('/a/file.zip', '.', '2.0.0')
```

Comment:
I guess we need to discuss here if this overwrites the already existing file or not.

Reply:
I switched to version 1.0.0